.storybook/themes/news.scss

```diff
@@ -459,6 +459,40 @@
 				),
 			),
 			blocks: (
+				header-nav-chain: (
+					background-color: var(--global-black),
+					height: 64px,
```

Comment:
this was hard-coded. but it could be a token clients set on mobile and desktop

Reply:
This was updated